icu4c/source/common/rbbi.cpp

```diff
@@ -1187,7 +1196,7 @@ static void U_CALLCONV initLanguageFactories() {
 }
 
 
-static const LanguageBreakEngine*
```

Comment:
this is surely wrong, LanguageBreakEngine should be immutable and not changed once created.

Reply:
Fixed